File: pat(advanced)C++/1005(advanced).cpp
#include<iostream>

using namespace std;
string convert[10] = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};

int main() {

    int index;
    int sum = 0;
    index = getchar();
    while (index != '\n') {
        sum += index - '0';
        index = getchar();
    };
    string output = to_string(sum);
    for (int i = 0; i < (int) output.length(); i++) {
        if (i != 0) {
            cout << ' ';
        }
        cout << convert[output[i] - '0'];
    }
    return 0;
}

File: pat(advanced)python/1005(advanced).py
if __name__ == '__main__':
    output = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'ten']
    line1 = input()
    sum1 = 0
    for i in line1:
        sum1 += int(i)
    flag = 0
    for i in str(sum1):
        if flag != 0:
            print(' ', end='')
        flag += 1
        print(output[int(i)], end='')
